cypress/integration/plugins/observability-dashboards/6_notebooks.spec.js

```diff
@@ -25,8 +27,25 @@ const moveToTestNotebook = () => {
     timeout: delayTime * 3,
   });
 
-  // Reload page to load notebooks if they are not flushed in OpenSearch index yet.
-  cy.reload();
+  // Force refresh the observablity index and reload page to load notebooks.
+  if (loadedOnce === 0) {
+    cy.request({
```

Comment:
It seems this refresh only execute once after `moveToNotebookHome` call. Is this by intentionally?